.packit.yaml

```diff
@@ -126,8 +126,6 @@ jobs:
         - artifacts:
           - type: repository-file
             id: https://copr.fedorainfracloud.org/coprs/g/cockpit/main-builds/repo/fedora-$releasever/group_cockpit-main-builds-fedora-$releasever.repo
-          - type: repository-file
-            id: https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/repo/fedora-$releasever/rhcontainerbot-podman-next-fedora-$releasever.repo
```

Comment:
I don't like this TBH -- seems to be prone to installing podman without matching dependencies? Isn't it often the case that newer podman versions need a newer crun/selinux policy etc.?

Your cited TF run is weird indeed. The `dnf update -y podman crun conmon criu` call is meant to upgrade to newer dependencies from the -next copr.  And e.g. crun is much newer, it has a "102:" epoch in COPR and no epoch in Fedora. I can debug this, but it's certainly *meant* to be used.

Or asked the other way around: Do you expect -next to be tested as a group and want to have non-released dependencies in podman? Or expect podman main to work on all released distros without newer deps?

Reply:
Ah, thanks for the explanation! So TF adds these extra repo files with a high prio number (i.e. low priority, ignored) by default? Ugh..

Reply:
In other words, adding the same fix to cockpit-podman.fmf sounds great!